SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -240,31 +255,35 @@ Interest InterestFactory::CreateServerWorkerInterest()
 	return ServerInterest;
 }
 
-Interest InterestFactory::CreateInterest() const
+Interest InterestFactory::CreateInterest(Worker_EntityId EntityId) const
 {
+	const auto Settings = GetDefault<USpatialGDKSettings>();
+	Interest ResultInterest;
 	if (Actor->IsA(APlayerController::StaticClass()))
 	{
-		return CreatePlayerOwnedActorInterest();
+		// Put the "main" interest queries on the player controller
+		AddPlayerControllerActorInterest(ResultInterest);
 	}
-	else if (GetDefault<USpatialGDKSettings>()->bEnableServerQBI)
+	if (Actor->GetNetConnection() != nullptr && Settings->bEnableClientResultTypes)
```

Comment:
So all these interests are valid to persist at the same time on an entity? If so can you add some spaces between the `if` statements.

Reply:
yep, that's the idea- will add